### PR TITLE
Add patch.estimates methods for GLEC-certified shipping estimates

### PR DESCRIPTION
**Pull request by holtbp**
### What
- Adds `patch.estimates.create_air_shipping_estimate` method
- Adds `patch.estimates.create_rail_shipping_estimate` method
- Adds `patch.estimates.create_road_shipping_estimate` method
- Adds `patch.estimates.create_sea_shipping_estimate` method

### Why
To provide GLEC-certified shipping estimates!

### SDK Release Checklist

- [ ] Have you added an integration test for the changes?
- [ ] Have you built the package locally and made queries against it successfully?
- [ ] Did you update the changelog?
- [ ] Did you bump the package version?
- [ ] If endpoints were removed, did you manually remove the corresponding files? (this should be rare)
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: test/integration/orders.test.js [hunk 1]
```diff
@@ -26,16 +26,16 @@ describe('Orders Integration', function () {
...
   });
 
-  it('supports creating an order with issuedTo', async function () {
-    const issuedTo = { email: 'issuee@companyc.com', name: 'Bob Dylan' };
+  it('supports creating an order with issued_to', async function () {
+    const issued_to = { email: 'issuee@companyc.com', name: 'Bob Dylan' };
     const { data } = await patch.orders.createOrder({
       total_price_cents_usd: 100,
-      issuedTo: issuedTo
+      issued_to: issued_to
```
Comment (kleinjm) [inline test/integration/orders.test.js:33]: was this a change we made to the API or SDK? Curious how this test was passing / working before

**Review by kleinjm — CHANGES_REQUESTED** [review]
LGTM other than the missing changelog entry. Thanks for updating the "issued to" tests as well

File: test/integration/orders.test.js (hunk 1 above)
Comment (holtbp) [reply test/integration/orders.test.js:33]: I'm also worried about this working previously. After generating the new code, this stopped working. But now that I remember, there was a support issue raised in Discord about this a while back: https://discord.com/channels/733029448558837792/754039768764907550/988853938356891718

**Review by kleinjm — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -77,11 +77,11 @@ const totalPrice = 500; // Pass in the total price in smallest currency unit (ie
...
 patch.orders.createOrder({ total_price: totalPrice, currency: currency });
 
-// Create order with the issuedTo field (optional)
+// Create order with the issued_to field (optional)
 const amount = 1_000_000; // Pass in the amount in unit specified
 const unit = 'g';
-const issuedTo = { email: 'issuee@companya.com', name: 'Olivia Jones' };
-patch.orders.createOrder({ amount: amount, unit: unit, issuedTo: issuedTo });
+const issued_to = { email: 'issuee@companya.com', name: 'Olivia Jones' };
+patch.orders.createOrder({ amount: amount, unit: unit, issued_to: issued_to });
```
Comment (pcothenet) [inline README.md:84]: Hum, for `node.js`, I'm 99% sure that the SDK actually requires `issuedTo`, not `issued_to`.

Comment (pcothenet) [reply README.md:84]: Yup: https://github.com/patch-technology/patch-node/blob/main/src/model/CreateOrderRequest.js#L74

Not a fan of the generator changing the case from the API parameter, but not sure it's worth fighting the generator for.

File: test/integration/orders.test.js (hunk 1 above)
Comment (pcothenet) [reply test/integration/orders.test.js:33]: I'm curious what changed this though. When I generated the code, it looked like the generator had turned the `issued_to` from the API into `issuedTo` for the SDK: https://github.com/patch-technology/patch-node/blob/main/src/model/CreateOrderRequest.js#L74

Can you create a PR for your changes to client-code-generation, I'd like to see what changed there.

**Review by pcothenet — CHANGES_REQUESTED** [review]
Good on the estimates, but I don't think issuedTo should be changed.

File: test/integration/orders.test.js (hunk 1 above)
Comment (pcothenet) [reply test/integration/orders.test.js]: Now that I look at it more closely though:

- `issuedTo` is the only parameter in camelCase, which isn't great
- but changing to `issued_to` would be a breaking change (e.g. for the person you answered to on [Discord](https://discord.com/channels/733029448558837792/754039768764907550/988853938356891718)

What's surprising to me is why the generation went with `issuedTo` for this one, and not the others 🤔

**Review by pcothenet — APPROVED** [review]
Thanks for keeping the changes separate!

File: README.md (hunk 2 above)
Comment (pcothenet) [inline README.md:84]: @holtbp looks like that got merged with this one. I'll modify the changelog to explain the breaking change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,11 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.24.0] - 2022-07-22
+
+### Added
+
+- Adds `patch.estimates.create_air_shipping_estimate` method
```
Comment (pcothenet) [inline CHANGELOG.md:12]: @holtbp it looks like it's `patch.estimates.createAirShippingEstimate` in node, no?

Comment (holtbp) [reply CHANGELOG.md:12]: You are correct

**Pull request merged**